modules/ROOT/pages/clustering/disaster-recovery.adoc

```diff
@@ -5,157 +5,292 @@
+First, ensure the `system` database is write available.
+Then, detach any potential lost servers from the cluster and replace them by new ones.
+Finish disaster recovery by starting or continuing to manage databases and verify that they are write available.
 
-. After the `system` database is available and the cluster's topology is satisfied, you can manage the databases.
+Every step consists of the following three sections:
 
-The steps are described in detail in the following sections.
+. A state that needs to be verified, with optional motivation.
+. An example of how the state can be verified.
+. A proposed series of steps to get to the correct state.
 
-[NOTE]
+[CAUTION]
 ====
-In this section, an _offline_ server is a server that is not running but may be _restartable_.
-A _lost_ server, however, is a server that is currently not running and cannot be restarted.
+Verifying each state before continuing to the next step, regardless of the disaster scenario, is recommended to ensure the cluster is fully operational.
 ====
 
+
+== Guide to disaster recovery
+
 [NOTE]
 ====
 Disasters may sometimes affect the routing capabilities of the driver and may prevent the use of the `neo4j` scheme for routing.
 One way to remedy this is to connect directly to the server using `bolt` instead of `neo4j`.
 See xref:clustering/setup/routing.adoc#clustering-routing[Server-side routing] for more information on the `bolt` scheme.
 ====
 
-=== Restore the `system` database
+=== Neo4j process started
 
-The first step of recovery is to ensure that the `system` database is available.
-The `system` database is required for clusters to function properly.
+==== State
+====
+The Neo4j process is started on all servers which are not _lost_.
+====
 
-. Start all servers that are _offline_.
-(If a server is unable to start, inspect the logs and contact support personnel.
-The server may have to be considered indefinitely lost.)
-. *Validate the `system` database's availability.*
-.. Run `SHOW DATABASE system`.
-If the response does not contain a writer, the `system` database is unavailable and needs to be recovered, continue to step 3.
-.. Optionally, you can create a temporary user to validate the `system` database's writability by running `CREATE USER 'temporaryUser' SET PASSWORD 'temporaryPassword'`.
-.. Confirm that the temporary user is created as expected, by running `SHOW USERS`, then continue to xref:clustering/disaster-recovery.adoc#recover-servers[Recover servers].
-If not, continue to step 3.
-+
-. *Restore the `system` database.*
-+
-[NOTE]
+==== Path to correct state
+Start the Neo4j process on all servers that are _offline_.
+If a server is unable to start, inspect the logs and contact support personnel.
+The server may have to be considered indefinitely lost.
+
+[[restore-the-system-database]]
+=== `System` database write availability
+
```

Comment:
This section is empty. Don't know if it's intentional because we're listing sth :/